src/figuresClasses.ts

```diff
@@ -1,19 +1,103 @@
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+enum Color {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue'
+}
+
+const MAIN_ERROR_MESSAGE = 'Invalid values';
+const ERROR_MESSAGE_TRIANGLE = `${MAIN_ERROR_MESSAGE}:
+  the longest side of a triangle is >= than a sum of two others`;
+const ERROR_MESSAGE_CIRCLE = `${MAIN_ERROR_MESSAGE}:
+  radius of a circle is smaller than 0`;
+const ERROR_MESSAGE_RECTANGLE = `${MAIN_ERROR_MESSAGE}:
+  width or height of a rectangle is smaller than 0`;
+
+const handleRound = (square: number): number => Math.floor(square * 100) / 100;
+const isTriangleValid = (a: number, b: number, c: number): boolean => {
```

Comment:
You called the variable `isTriangleValid`, but in the middle you check only negative conditions, so this check should be called `isTriangleNotValid`.